### DISCUSSION TITLE
Same hook like nuxtServerInit in Nuxt3

### DISCUSSION THREAD

**Discussion by k0mar12** [nuxt/nuxt#18094] [Questions] [answered]
Hey there.

As i can see, vuex has been removed from Nuxt3, so question next

Is exist any method to create some equal hook like nuxtServerInit ?

**Comment by wobsoriano** [comment, marked as answer]
https://github.com/nuxt/framework/discussions/1915

**Reply by k0mar12** [reply]
Got it, thx.

One more question, i created in plugins init.server.js and it firing twice on server side, what it could be?
I know about same behavior in nuxt2 and how to fix it, but what about nuxt3 ?

**Reply by psycongaroo** [reply]
> Got it, thx.
> 
> One more question, i created in plugins init.server.js and it firing twice on server side, what it could be? I know about same behavior in nuxt2 and how to fix it, but what about nuxt3 ?

Check favicon

**Reply by k0mar12** [reply]
Thank You so much, I would never have guessed :)

**Reply by psycongaroo** [reply]
No problem, if you want know why:

nuxt/nuxt#16281
